### PR TITLE
Minor Update | Readme | Contribution.md

### LINKED ISSUE #29: update README to include a docs usage and add a contribution file in root
Please adhere to this format when creating or customizing ARC modules

https://github.com/sourcefuse/terraform-aws-arc-github-iam/blob/main/docs/module-usage-guide/README.md

Please include a  [CONTRIBUTING.md](https://github.com/sourcefuse/terraform-aws-arc-github-iam/blob/main/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by vijay-stephen**
#### Description

Github issue: [issue 83](https://github.com/sourcefuse/terraform-aws-arc-bootstrap/issues/29)

#### Objective

Please adhere to this format when creating or customizing ARC modules

https://github.com/sourcefuse/terraform-aws-arc-github-iam/blob/main/docs/module-usage-guide/README.md

Please include a [CONTRIBUTING.md](https://github.com/sourcefuse/terraform-aws-arc-github-iam/blob/main/CONTRIBUTING.md)

#### Type

- [ ] Bug Fix
- [ ] New Feature
- [ ] Performance Tweaks
- [ ] Style Tweaks
- [x] Code Refactoring
- [x] Documentation
- [ ] Tests

#### Work Done

- Add usage guide and CONTRIBUTING.md
- update README.md
- Removed deprecated code and replaced the same with dedicated resource blocks

### CODE REVIEW

File: .github/workflows/update-docs.yml [hunk 1]
```diff
@@ -12,15 +12,41 @@ jobs:
...
           API_TOKEN_GITHUB: ${{ secrets.ARC_DOCS_API_TOKEN_GITHUB }}
         with:
           source_file: 'README.md'
           destination_repo: 'sourcefuse/arc-docs'
-          destination_folder: 'docs/arc-iac-docs/modules/terraform-module-aws-bootstrap'
+          destination_folder: 'docs/arc-iac-docs/modules/terraform-aws-ref-arch-bootstrap'
+          user_email: 'github-actions@github.com'
+          user_name: ${{ github.actor }}
+          commit_message: ${{ github.event.head_commit.message }}
+      - name: Pushes static folder
```
Comment (mayank0202) [inline .github/workflows/update-docs.yml:25]: @vijay-stephen Static folder is not there in the repo then there is no need to add this workflow for pushing static folder

Comment (vijay-stephen) [reply .github/workflows/update-docs.yml:25]: let me update it, it has image folder

File: README.md [hunk 2]
```diff
@@ -83,58 +195,14 @@ No modules.
...
-  If you want to migrate a select number of workspaces, you must manually
-  pull and push those states.
-
-  If you answer "yes", Terraform will migrate all states. If you answer
-  "no", Terraform will abort.
-```
-
-Our local state has now been migrated to the new backend. It is now safe to remove the local `terraform.tfstate`.
-
 ## Development
```
Comment (vijay-stephen) [inline README.md:206]: @tsaucier-sf shall we remove this section from Readme, as its not in Contribution.md

**Review by mayank0202 — APPROVED** [review]

File: .github/workflows/update-docs.yml (hunk 1 above)
Comment (mayank0202) [reply .github/workflows/update-docs.yml:25]: Okay

**Review by tsaucier-sf — CHANGES_REQUESTED** [review]
Be sure to update `ref-arch` to `arc`

**Comment by vijay-stephen** [comment]
> Be sure to update `ref-arch` to `arc`

checked, its not there in any files

```
terraform-aws-arc-bootstrap > grep -Ril "refarch" *
example/.terraform/modules/tags/mkdocs.yml
example/.terraform/modules/tags/README.md
example/.terraform/modules/tags/.github/workflows/update-docs.yml
example/.terraform/modules/tags/catalog-info.yaml
```

**Review by tsaucier-sf — APPROVED** [review]

**Pull request merged**